src/autoloads/connections.gd

```diff
@@ -0,0 +1,118 @@
+extends Node
+
+enum connectionTypes {
+	LOCAL,				# Local only game, tutorial
+	DEDICATED_SERVER,	# Server only, no local client
+	CLIENT_SERVER,		# Server with a local player
+	CLIENT				# Client only, remote server
+}
+
+var connectionType: int = connectionTypes.LOCAL setget toss, getConnectionType
+var myName: String = "" setget toss, getMyName
+var serverName: String = "" setget toss, getServerName
+const MAX_PLAYERS = 20
+var listConnections = {}
```

Comment:
Could explicitly set the type of these variables just for consistency with other scripts